spec/steps/prepare.fmf

```diff
@@ -27,6 +27,44 @@ description: |
         - /tmt/steps/provision/vagrant
         - /tmt/steps/provision/testcloud
 
+/install:
+    summary:
+        Install a package using dnf/yum.
+    description:
+        Installs a package(s) from specified source and verifies that it was
+        installed successfuly.
+    example: |
+        prepare:
+            how: install
+            package: ruby
+    implemented:
+        - /tmt/steps/prepare
+
+/install-copr:
+    summary:
+        Enable COPR repository using dnf and install package(s).
+    description:
+        Enable COPR repository and then run same as install.
+    example: |
+        prepare:
+            how: install
+            copr: pvalena/vagrant
+            package: vagrant
+    implemented:
+        - /tmt/steps/prepare
+
+/install-koji:
+    summary:
+        Download build package(s) and install them.
+    description:
+        Downloads build packages from koji (or brew) and install them using dnf/yum.
+    example: |
+        prepare:
+            how: install
+            url: https://koji.fedoraproject.org/koji/buildinfo?buildID=1465484
```

Comment:
The `url` sounds very generic to me. You can install rpm packages directly from and `url` as well. What about calling the key `build`? This would a bit more specific and works for both koji and brew. We could also support fetching builds by nvr or id in the future, what do you think?

Reply:
Could it be all three build/task/tag ? E.g current example would be `build = 1465484`. 
When I need exact nvr and if we are ok with automatic nvr/id detection it would be `build = foo-1.f31`.
At the moment 'tag' might be useful for module scratch builgs `tag = name_of_tag`
Task is similar to build but accepts only id.

I'm not saying everything should be possible from the beginning, just possibility is enough for now.

Reply:
@psss sure, `build` is fine with me.

@lukaszachy it actually does parse the URI and gets just the build number from it. Then it uses `$ koji ...` or `$ brew`, based on the host name, so we'd have to take that into account too. Opting for simple "build url" at first seemed best to me (the options are limitless off course).
Would build URL be useful, as a start? (Note that blacklisting install packages is ortogonal to this.)